app/frontend/src/store.ts

```diff
@@ -59,6 +59,6 @@ export type AppDispatch = typeof store.dispatch;
 export type AppThunk = ThunkAction<void, RootState, null, Action<string>>;
 
 export const useAppDispatch = () => useDispatch<AppDispatch>();
-export const useTypedSelector: TypedUseSelectorHook<RootState> = useSelector;
+export const useTypedSelector = createSelectorHook<RootState>();
```

Comment:
Switched to using `createSelectorHook` as I was told by the editor `TypedUseSelectorHook` is deprecated.